polars/dataframe.ts

```diff
@@ -1707,6 +1704,55 @@ function map(df: DataFrame, fn: (...args: any[]) => any[]) {
   return df.rows().map(fn);
 }
 
+function mapPolarsTypeToJSONSchema(colType: DataType): string {
+  const typeMapping: { [key: string]: string } = {
+    Null: "null",
+    Bool: "boolean",
+    Int8: "integer",
+    Int16: "integer",
+    Int32: "integer",
+    Int64: "integer",
+    UInt8: "integer",
+    UInt16: "integer",
+    UInt32: "integer",
+    UInt64: "integer",
+    Float32: "number",
+    Float64: "number",
+    Date: "string",
+    Datetime: "string",
+    Utf8: "string",
+    Categorical: "string",
+    List: "array",
+    Struct: "object",
+  };
+
+  const dataType = colType.variant;
+  return typeMapping[dataType] || "string";
+}
+
+/////// HTML ESCAPING ///////
+const rawToEntityEntries = [
```

Comment:
Any reason this is a global const but only used inside of `escapeHTML` func?
Should we move it inside of the function?

Reply:
Style wise I understand. However this will create it on every call, which is called for _every_ cell in the table output.

I think there's an inner dependency that relies on `escape-html` I could add in as an explicit dependency if that's better.